bricks/template/__brick__/{{project_name.snakeCase()}}/.github/workflows/ios_deploy_to_app_store.yml

```diff
@@ -19,37 +19,70 @@ jobs:
       FASTLANE_SESSION: ${{#mustacheCase}}secrets.FASTLANE_SESSION{{/mustacheCase}}
       MATCH_PASSWORD: ${{#mustacheCase}}secrets.MATCH_PASSWORD{{/mustacheCase}}
       KEYCHAIN_PASSWORD: ${{#mustacheCase}}secrets.KEYCHAIN_PASSWORD{{/mustacheCase}}
-      GITHUB_RUN_NUMBER: $secrets.GITHUB_RUN_NUMBER
+      APP_STORE_KEY_ID: ${{#mustacheCase}}secrets.APP_STORE_KEY_ID{{/mustacheCase}}
+      APP_STORE_ISSUER_ID: ${{#mustacheCase}}secrets.APP_STORE_ISSUER_ID{{/mustacheCase}}
+      APP_STORE_CONNECT_API_KEY_BASE64: ${{#mustacheCase}}secrets.APP_STORE_CONNECT_API_KEY_BASE64{{/mustacheCase}}
+      GITHUB_RUN_NUMBER: $GITHUB_RUN_NUMBER
     steps:
-    - name: Check out
-      uses: actions/checkout@v3
+      - name: Check out
+        uses: actions/checkout@v3
 
-    - name: Install SSH key
-      uses: webfactory/ssh-agent@v0.4.1
-      with:
-        ssh-private-key: ${{#mustacheCase}}secrets.SSH_PRIVATE_KEY{{/mustacheCase}}
+      - name: Install SSH key
+        uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{#mustacheCase}}secrets.SSH_PRIVATE_KEY{{/mustacheCase}}
 
-    - name: Set up Flutter environment
-      uses: subosito/flutter-action@v2
-      with:
-        channel: 'stable'
-        flutter-version: '3.10.5'
+      - name: Set up Flutter environment
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+          flutter-version: '3.10.5'
 
-    - name: Get Flutter dependencies
-      run: flutter pub get
+      - name: Get Flutter dependencies
+        run: flutter pub get
 
-    - name: Run code generator
-      run: flutter packages pub run build_runner build --delete-conflicting-outputs
+      - name: Set up Production env
+        env:
+          ENV: ${{#mustacheCase}} secrets.ENV {{/mustacheCase}}
```

Comment:
Please keep an eye on [my suggestion](https://github.com/nimblehq/flutter-templates/pull/265#discussion_r1309594430). There are no spaces between braces to avoid [double spaces generating](https://github.com/nimblehq/flutter-templates/pull/265/files#diff-11df84cbee4a217a49706cc4b5ca4c3c37a883e94a43aba1e18e8af4474a9d6cR44-R48) 🙏 

![image](https://github.com/nimblehq/flutter-templates/assets/16315358/1980fc3a-7f65-43e2-b085-ddcee7dda295)


```suggestion
          ENV: ${{#mustacheCase}}secrets.ENV{{/mustacheCase}}
```